packages/mui-material/src/index.d.ts

```diff
@@ -488,5 +488,5 @@ export * from './generateUtilityClass';
 
 export { default as generateUtilityClasses } from './generateUtilityClasses';
 
-export { default as Unstable_TrapFocus } from './Unstable_TrapFocus';
-export * from './Unstable_TrapFocus';
+export { default as Unstable_FocusTrap } from './Unstable_FocusTrap';
+export * from './Unstable_FocusTrap';
```

Comment:
```suggestion
```

I think we can remove this module altogether. It's unstable, no longer documented, and we encourage to use https://mui.com/base-ui/react-focus-trap/ instead now. 

Now, this should first happen in MUI X, so I don't think there is much to do on Material UI now. I think we can close this PR.